subsys/net/lib/nrf_cloud/src/nrf_cloud_fota.c

```diff
@@ -682,6 +683,8 @@ static void http_fota_handler(const struct fota_download_evt *evt)
 	switch (evt->id) {
 	case FOTA_DOWNLOAD_EVT_FINISHED:
 		LOG_INF("Download complete");
+		nrf_cloud_download_end();
```

Comment:
sure, it would be possible, but it would require much more code.
this was the simplest way.
if more advanced management of nrf_cloud downloads are required in the future, this could be implemented.